configure.ac

```
@@ -2829,7 +2829,11 @@ AC_CHECK_ALIGNOF(int16_t)
 AC_CHECK_ALIGNOF(int32_t)
 AC_CHECK_ALIGNOF(int64_t)
 AC_CHECK_ALIGNOF(bool)
-AC_CHECK_ALIGNOF(wchar_t, 0, [#include <stddef.h>])
+AC_CHECK_ALIGNOF(wchar_t, [
+#ifdef HAVE_STDDEF_H
+#include <stddef.h>
+#endif
+])
```

Your fix is good, but I think it doesn't have to be so complicated. The problem with the original code is that is misuses the AC_CHECK_ALIGNOF macro.
```
AC_CHECK_ALIGNOF (type, [includes = ‘AC_INCLUDES_DEFAULT’])
```
The second argument `0` makes no sense and presumably wipes out the third. I think the fix can be simplified to
Your fix is good, but I think it can be simplified. The problem with the original code is that is misuses the AC_CHECK_ALIGNOF macro.
```
AC_CHECK_ALIGNOF (type, [includes = ‘AC_INCLUDES_DEFAULT’])
```
The second argument `0` makes no sense and whatever is in the third looks to be discarded. I think the fix can be simplified to
```suggestion
AC_CHECK_ALIGNOF(wchar_t)
```
According to the documentation (https://www.gnu.org/software/autoconf/manual/autoconf-2.67/html_node/Generic-Compiler-Characteristics.html) when no includes argument is given, then the default includes will be used. stddef.h is included in the default includes https://www.gnu.org/software/autoconf/manual/autoconf-2.67/html_node/Default-Includes.html#Default-Includes.

---

ok, that works, thanks.